### PR TITLE
Fix CSS On Stats Page From Bootstrap 4 Upgrade

### LINKED ISSUE #5728: Stats page bug
A couple of things looking off on the stats page https://publiclab.org/stats. Font a little huge, back and front arrows not aligned correctly plus wierd gray background, needs more spacing between the sections, question asked vs question answered bar graph not showing one side percentage

### PR DESCRIPTION
**Pull request by skilfullycurled**
Fixes #5728

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [x] ask `@publiclab/reviewers` for help, in a comment below

### CODE REVIEW

**Comment by cesswairimu** [comment]
Hey @skilfullycurled, yeah that should not be failing :thinking: . Restarting the build to see if anything changes

**Comment by cesswairimu** [comment]
Seems like that fixed it

**Comment by skilfullycurled** [comment]
@cesswairimu, thank you!  What does "restarting the build process mean"?  

Also, when I attach a screenshot(s) do I need to point out each change (say, with a red arrow), or just show a before and after comparison?  Some of the changes are quite subtle (h5's to h6's, for example) though I suppose the diffs will show where they are...?

Going to make the changes discussed re: centering the text and I'll be back for review.

**Comment by cesswairimu** [comment]
@skilfullycurled restarting the build is hitting the restart button on the top right of build log

Screenshot from 2019-05-29 19-03-47

As for screenshot a screenshot of the /stats page should be enough. Thanks.

**Comment by skilfullycurled** [comment]
Ah, thanks, great tip!

**Comment by jywarren** [comment]
We've had some instability from our new system tests (although they're overall awesome!) So this may have caused the brief test failure. Glad to see it was resolved! And, we usually just ask for simple screenshots, so no need to annotate unless you want to!

This sounds like it'll be a great change, thanks so much to both of you!

**Comment by skilfullycurled** [comment]
Oh, forgot to upload image.  The whole page is too large to take an image of but everything below is unchanged anyway.

Screen Shot 2019-06-07 at 5 37 56 PM

**Comment by skilfullycurled** [comment]
@publiclab/reviewers, this PR is all set and ready for your approval!

File: app/views/stats/_range.html.erb [hunk 1]
```diff
@@ -110,12 +106,12 @@
 </div>
     <br><br> <br>
 <% end %>
+<h6> Move back or forward using the buttons below </h6>
  <div class="row">
-<h5> Move back or forward using the buttons below </h5>
- <div class="col-lg-4 col-lg-offset-1">
+ <div class="col-lg-4 offset-col-lg-1">
```
Comment (cesswairimu) [inline app/views/stats/_range.html.erb]: Hey @skilfullycurled lets change this to `<div class="col-lg-4 offset-lg-1">`

File: app/views/stats/index.html.erb [hunk 2]
```diff
@@ -1,16 +1,16 @@
-<div class="col-lg-12 col-lg-offset-1" style="margin-bottom:20px; margin-left:25px;">
+<div class="col-lg-12 offset-col-lg-1" style="margin-bottom:20px; margin-left:25px;">
```
Comment (cesswairimu) [inline app/views/stats/index.html.erb]: `<div class="col-lg-12 offset-lg-1"`

File: app/views/stats/index.html.erb [hunk 3]
```diff
@@ -1,16 +1,16 @@
...
+                     :time => time_ago_in_words(Node.find_by(type: 'note', status: 1).created_at)) %></b></h5>
     <br>
 
     <div class="row">
         <div class=col-lg-6>
-            <h5> Statistics from <%= @start.to_formatted_s(:long)%> to <%= @end.to_formatted_s(:long)%> </h5>
+            <h6> Statistics from <%= @start.to_formatted_s(:long)%> to <%= @end.to_formatted_s(:long)%> </h6>
         </div>
         <div class=col-lg-2>
             <%= form_tag request.url, method: 'get', enforce_utf8: false do %>
```
Comment (cesswairimu) [inline app/views/stats/index.html.erb]: also on line 29 lets change to  `<div class="col-md-4 offset-md-1">`.

**Comment by skilfullycurled** [comment]
Will do @cesswairimu, one I have a clarifying question.  

> Also any other places I missed with offset lets have them in the new format. 

Issue #5728 had two patterns but given what you have asked for above, I am interpreting the "new format" to mean that in any place where there is _both_ an `offset` _and_ a `col`, the `offset` should be at the beginning, and the `col` should be removed so that the pattern is `offset-size-num`

Examples (the last two of which I'm not sure exist):

a) `col-lg-offset-1` -> `offset-lg-1`
b) `offset-col-lg-1` -> `offset-lg-1`
c) `col-md-offset-1` -> `offset-md-1`
c) `offset-col-md-1` -> `offset-md-1`
d) `col-sm-offset-3` -> `offset-sm-3`
e) `offset-sm-col-3` -> `offset-sm-3`

Do I have that right?

**Comment by cesswairimu** [comment]
Yeah that's right, my bad I made a mistake and include col in one. As for the last two I am talking about the line https://github.com/publiclab/plots2/blob/master/app/views/stats/_range.html.erb#L33 and https://github.com/publiclab/plots2/blob/master/app/views/stats/_range.html.erb#L115. Thanks

**Comment by skilfullycurled** [comment]
Thanks @cesswairimu!  Again couldn't have done it with out you, especially since it was CSS.  It was big win for me to change the alignment of an element and not have the entire page fall apart!  

I think I need one more person to give approval?  So, giving a mention here to any @publiclab/reviewers to approve at their leisure!

**Comment by jywarren** [comment]
This looks really lovely! Thanks @skilfullycurled for this great work! And thanks @cesswairimu for supporting!!!! Great teamwork! 🎉 👍 🙌

**Pull request merged**